setup.py

```diff
@@ -123,6 +123,7 @@
     extras_require=EXTRAS_REQUIRE,
     entry_points={
         "console_scripts": [
+            "bbs_database = bluesearch.entrypoint.database.parent:main",
```

Comment:
> I guess this choice was just to be in line with the other entrypoint names, so fair enough.

Not against using `-` instead of `_`, but I would be for doing that in a separate PR, and keep it consistent as Jan did here for the moment.

Reply:
yep, agree about separate PR